src/main/java/xbot/common/controls/actuators/XCANMotorController.java

```diff
@@ -31,12 +32,25 @@ XCANMotorController create(
     protected XCANMotorControllerInputsAutoLogged inputs;
     protected String akitName;
 
+    protected boolean usesPropertySystem = true;
```

Comment:
⭐ ⭐ could you add comments about what these do and when to change them?

Reply:
Added in a followup commit! :)